config/webpack.config.babel.js

```diff
@@ -98,7 +100,7 @@ export default {
   externals: [filterLocales],
   stats: 'minimal',
   target: 'web',
-  devtool: production ? false : 'eval-source-map',
+  devtool: production ? 'source-map' : 'eval-source-map',
```

Comment:
I see now that we produce source files for all JS and CSS files in the output, including all of the locale files. I'm sure there's a way to limit this to a whitelist only (should be just the main bundle's JS and CSS).

Reply:
I don't think we should produce sourcemaps for css files as well, it's not useful for sentry